src/data/mv_polynomial/basic.lean

```diff
@@ -364,6 +364,11 @@ begin
   case h_X : p i hp { exact S.mul_mem hp (algebra.subset_adjoin $ mem_range_self _) }
 end
 
+@[ext] lemma linear_map_ext {M : Type*} [add_comm_monoid M] [module R M]
+  {f g : mv_polynomial σ R →ₗ[R] M} (h : ∀ s, f ∘ₗ monomial s = g ∘ₗ monomial s) :
+  f = g :=
+by { ext1, apply h }
```

Comment:
Can you also show this with something like:
```suggestion
add_monoid_algebra.linear_map_ext h
```
That feels slightly nicer as it reveals what's actually going on.